Rectangle/mul.lproj/Main.xcstrings

```diff
@@ -1778,7 +1802,7 @@
       }
     },
     "2oI-Rn-ZJC.title" : {
-      "comment" : "Class = \"NSMenuItem\"; title = \"Transformations\"; ObjectID = \"2oI-Rn-ZJC\";",
+      "comment" : "Class = \"NSMenuItem\"; title = \"c\"; ObjectID = \"2oI-Rn-ZJC\";",
```

Comment:
By default, this menu item doesn't exist in the app, but this looks like a typo?